SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -136,3 +136,29 @@ void CDriver::BoundaryConditionsUpdate() {
     geometry_container[iZone][INST_0][MESH_0]->UpdateCustomBoundaryConditions(geometry_container[iZone][INST_0],config_container[iZone]);
   }
 }
+
+////////////////////////////////////////////////////////////////////////////////
+/* Functions related to dynamic mesh */
+////////////////////////////////////////////////////////////////////////////////
+
+void CDriver::SetTranslationRate(passivedouble xDot, passivedouble yDot, passivedouble zDot) {
+  if (rank == MASTER_NODE) {
+    cout << endl << " Setting new translational velocity in x, y, z direction.";
+  }
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/su2code/SU2/commit/4f00bcce1bf5b7eb42c6aafcb9f5dff9ead4dbd6